apps/editing-toolkit/editing-toolkit-plugin/premium-content/blocks/container/stripe-nudge.js

```diff
@@ -64,6 +64,8 @@ export default compose( [
 		 * Complicated to define the valid type with JSDoc
 		 *
 		 * @param dispatch
+		 * @param root0
+		 * @param root0.stripeConnectUrl
```

Comment:
I'd probably remove the changes from these two non-newspack files if you can, just to keep things simpler!

Reply:
Thanks  @noahtallen , updated the branch